Jenkinsfile

```diff
@@ -157,7 +157,7 @@ pipeline {
                                 cd performance-tests-cmdstan
                                 cd cmdstan; make -j${env.PARALLEL} build; cd ..
                                 cp ../bin/stanc cmdstan/bin/stanc
-                                ./runPerformanceTests.py -j${env.PARALLEL} --runs=0 ../test/integration/good
+                                ./runPerformanceTests.py -j7 --runs=0 ../test/integration/good
```

Comment:
This is because with the default parallel setting the instances run out of RAM, with -j7 the tests pass but they run for 3 hours instead of 20-30mins! We need to get to the bottom of this. 

Does this effect "typical" models or is it just a factor with these huge test models ala [this one](https://github.com/stan-dev/stanc3/blob/master/test/integration/good/function-signatures/distributions/univariate/continuous/skew_normal/skew_normal_ccdf_log_1.stan).

Reply:
Yeah I think this is just with the huge models. Though @rok-cesnovar I think you and I have both tried these locally without issue. It could be the compiler we use for running these tests? I'm really not sure. My guess as to what's happening is that we are now generating signatures for both `Eigen::Map<>` types and `Eigen::Matrix<>` types

Reply:
I havent explored it for this PR yet, but have it on my list for the next few days.